go.mod

```diff
@@ -18,77 +17,74 @@ require (
 	github.com/prometheus/client_golang v1.9.0 // indirect
 	github.com/prometheus/common v0.19.0 // indirect
 	github.com/prometheus/procfs v0.6.0 // indirect
-	golang.org/x/crypto v0.0.0-20210317152858-513c2a44f670 // indirect
-	golang.org/x/net v0.0.0-20210316092652-d523dce5a7f4 // indirect
+	golang.org/x/net v0.0.0-20210410081132-afb366fc7cd1 // indirect
 	golang.org/x/oauth2 v0.0.0-20210313182246-cd4f82c27b84 // indirect
-	golang.org/x/sys v0.0.0-20210317225723-c4fcb01b228e // indirect
 	golang.org/x/term v0.0.0-20210317153231-de623e64d2a6 // indirect
-	golang.org/x/text v0.3.5 // indirect
-	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba // indirect
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/genproto v0.0.0-20210317182105-75c7a8546eb9 // indirect
 	google.golang.org/grpc v1.36.0
-	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	k8s.io/api v0.21.0-beta.1
-	k8s.io/apimachinery v0.21.0-beta.1
-	k8s.io/client-go v1.5.2
-	k8s.io/component-base v0.20.4 // indirect
-	k8s.io/csi-translation-lib v0.21.0-beta.1
+	k8s.io/api v0.21.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/client-go v0.21.0
+	k8s.io/component-base v0.21.0 // indirect
+	k8s.io/csi-translation-lib v0.21.0
 	k8s.io/klog/v2 v2.8.0
 	k8s.io/kube-openapi v0.0.0-20210305164622-f622666832c1 // indirect
 	k8s.io/utils v0.0.0-20210305010621-2afb4311ab10 // indirect
-	sigs.k8s.io/structured-merge-diff/v4 v4.1.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.1.1 // indirect
 )
 
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.0
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.21.0
```

Comment:
same question as resize PR. I think we don't need these replace statements if we're not importing k8s.io/kubernetes

Reply:
I have less faith in that. But we can try without them and then we'll see whether the default version selection for indirect dependencies will do a good enough job of selecting a consistent set of module versions.

Reply:
I tried it out on my local workspace and it seems fine?

Reply:
Fixed with the new release-tools.